deploy/release.py

```diff
@@ -0,0 +1,66 @@
+import helpers
+import os
+import argparse
+import sys
+
+DATABASE = os.environ.get("OPSCENTER_DATABASE", "DEV")
+SCHEMA = "CODE"
+APPLICATION_PACKAGE = os.environ.get("OPSCENTER_PACKAGE", "OPSCENTER")
+APPLICATION = os.environ.get("OPSCENTER_APP", "OPSCENTER_APP")
+STAGE = "SD"
+FULL_STAGE = f"@{DATABASE}.{SCHEMA}.{STAGE}"
+FULL_STAGE_SLASH = FULL_STAGE + "/"
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        description="Release a new version of the application package"
+    )
+    parser.add_argument(
+        "-p", "--profile", help="Snowflake profile to use", default="opscenter"
+    )
+    parser.add_argument(
+        "-u",
+        "--update",
+        action="store_true",
+        help="Update the application package and start a scan",
+    )
+    parser.add_argument(
+        "-r",
+        "--release",
+        help="Release the application package",
+        default="DEFAULT",
+        nargs="*",
+    )
+    parser.add_argument(
+        "-x", "--patch", help="Patch of the application package to release"
+    )
+
+    args = parser.parse_args()
+    if args.update and args.release:
+        print("Cannot update and release at the same time")
+        sys.exit(1)
+    conn = helpers.connect_to_snowflake(args.profile)
+    cur = conn.cursor()
+    if args.update:
+        cur.execute(
+            f"ALTER APPLICATION PACKAGE {APPLICATION_PACKAGE} ADD PATCH FOR VERSION v2 USING {FULL_STAGE_SLASH}"
```

Comment:
nit: we'll eventually need to come back and add an argparse argument for the version.